packages/demo-cards/claim.gts

```diff
@@ -290,6 +292,39 @@ class Isolated extends Component<typeof Claim> {
   }
 }
 
+class Embedded extends Component<typeof Claim> {
+  get cannotClickViewClaimButton() {
```

Comment:
Maybe `claimButtonDisabled` is a simpler name for this?

Reply:
Resolved https://github.com/cardstack/boxel/pull/418/commits/e71162a3c1bcc0df94a3870ac235dc6b5829696d